udrzba/_pomocne.php

```diff
@@ -33,7 +33,7 @@ function nasad($nastaveni) {
 
       /web/soubory/*
       !/web/soubory/styl
-      !/web/soubory/*.js
+      !/web/soubory/*.*
```

Comment:
U tohohle jsem trochu nervózní, aby nás to do budoucna nepřekvapilo a nenasadil se soubor, se kterým nepočítáme - nedáme tam striktněji jen js + html?
`!/web/soubory/*.js`
`!/web/soubory/*.html`

Reply:
Upravil jsem to tak. Puvodni myslenka byla, ze by toto kopirovalo gitignore (kde to dava smysl).